app/controllers/members_controller.rb

```diff
@@ -20,11 +20,6 @@ def toggle_manager
   def toggle_active
     find_member
     @member.toggle(:active).save!
-    if @member.active
-      @member.add_all_posts_to_index
-    else
-      @member.remove_all_posts_from_index
-    end
```

Comment:
@sseerrggii are we sure we don't need this?

Reply:
Maybe we're covered by https://github.com/coopdevs/timeoverflow/pull/465/files#diff-ab4e9273f7df75aa5004a42f757d6f51R7 but we should test both cases just to make sure:

1. setting a user as `active`, will her old posts show up now in search results?
2. setting a user ar `inactive`, will her posts be hidden in search results?

Reply:
Yes, le'ts test it. I'm not sure the behavior without Elastic Search but we need to show only Post from active Members

Reply:
Done in https://github.com/coopdevs/timeoverflow/pull/465/commits/5d0ec9ee9e03c6b732cbc3d6a58d20d7653ba31b